connection.js

```diff
@@ -580,7 +611,7 @@ class Connection {
             // this avoids TimecontrolString being frozen on the same value independently from what user chooses, 
             // e.g. stuck on "absolute"  
 
-            // for fischer, byoyomi, or canadian, we use our UHMAEAT !
+            // for fischer, byoyomi, canadian, simple, we use our UHMAEAT !
```

Comment:
Universal Highly Modulable And Expandable Argv Tree, see : https://github.com/wonderingabout/gtp2ogs/blob/bigToSmallPR-Part1/connection.js#L438

Reply:
another function would be on top of #129 if you or someone wants to do it, it would be a great enhancement

but since UHMAEAT 2.2 already works, i'd rather merge it for the enhancements it brings

also since its such a special thing that is not fast to maintain and i've got a liking to it, i'd like to keep its name